src/main/java/frc/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -170,11 +170,11 @@ public DrivebaseSubsystem() {
     swerveModules = // modules are always initialized and passed in this order
         new SwerveModule[] {frontRightModule, frontLeftModule, backLeftModule, backRightModule};
 
-    rotController = new PIDController(.0175, 0, 0.0015);
+    rotController = new PIDController(0.04, 0.001, 0.003);
```

Comment:
would like to see constants moved to constant file definitions, perhaps in a little structure to make them obviously related